### ISSUE TITLE
Add references to manpages.opensuse.org

### ISSUE THREAD

**Issue by thkukuk** [openSUSE/landing-page#222]
https://manpages.opensuse.org is up.

We need to add references to it on www.opensuse.org and maybe in to the big icon in the upper right corner?
(Between, we should replace Kubic with MicroOS there).

**Comment by thkukuk** [comment]
@hellcp @ddemaio I think something for you

**Comment by ddemaio** [comment]
We can look at it @thkukuk. There is very little other than maybe replacing kiwi with it under tools. We don't have any reference to Kubic on opensuse.org. We just put up MicroOS just recently. BTW, I'll start putting together something for news.o.o about manpages.o.o.

**Comment by thkukuk** [comment]
The menu of the chameleon style as we use it on https://microos.opensuse.org, https://manpages.opensuse.org/, https://doc.opensuse.org/, ... so everywhere except for www.opensuse.org itself has a big button in the upper right corner, and there the second entry in the "Other" section is "Kubic", which needs to be replaced with "MicroOS".
Sorry if there is a better way to report that, I don't understand where to report what, that's really confusing with the openSUSE infrastructure...

**Comment by hellcp** [comment]
Indeed, the megamenu is a part of https://github.com/opensuse/chameleon, but I am currently working on splitting that out of the theme and putting it up on a website instead, so that we have a static version of it too. It will also make it easier to update the links there in the future as well.

**Comment by hellcp** [comment]
I have started making some moves to make this easier, https://github.com/openSUSE/universe-o-o/pull/1 creates universe.opensuse.org site, that is gonna be the place which will gather all the links within our infrastructure, and https://github.com/openSUSE/chameleon/pull/47 migrates the chameleon theme to use the new universe api provided by that site. The next thing to do would be to finish replacing this repo with www-o-o, but that still requires some work
